plugins/inara.py

```diff
@@ -189,7 +199,7 @@ def journal_entry(cmdr, is_beta, system, station, entry, state):
 
     # Send any unsent events when switching accounts
     if cmdr and cmdr != this.cmdr:
-        call()
+        call(force=True)
```

Comment:
But, yes, this force is fine as Cmdr changes, and thus should be using a different Inara API key anyway, so won't cause issues.

In *theory* someone could be on CmdrA and have CmdrB in a different install sat in the launcher ready to go, but I still think they'd not be able to actually get in-game within 30s to bust the flood limit.  If they're using different Windows users for this then they need to do that also for EDMC anyway, so no issue.